tox.ini

```diff
@@ -148,7 +150,8 @@ commands_pre=
     pip install -r {envtmpdir}/requirements.txt
     pip freeze
 commands=
-    pytest asdf-astropy
+    pytest asdf-astropy \
+        -W "ignore::asdf.exceptions.AsdfDeprecationWarning:asdf.types:504"
```

Comment:
This will be fixed by the above request on the entry point.

Reply:
I removed this line.